db/seeds.rb

```diff
@@ -6,11 +6,28 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-db = Specialty.create(name: "Database", url_name: "database", image: File.new("#{Rails.root}/app/assets/images/database/horse_animal_totem.png"))
-fe = Specialty.create(name: "Front End", url_name: "front_end", image: File.new("#{Rails.root}/app/assets/images/back_end/Wolf_animal_totem.png"))
-be = Specialty.create(name: "Back End", url_name: "back_end", image: File.new("#{Rails.root}/app/assets/images/front_end/hawk_animal_totem.png"))
+db = Specialty.create(name: "Database",
+        url_name: "database",
+        image: File.new("#{Rails.root}/app/assets/images/database/horse_animal_totem.png"))
```

Comment:
Line is too long. [91/80]<br>Align the elements of a hash literal if they span more than one line.